src/v2/Components/UserSettings/TwoFactorAuthentication/Components/AppSecondFactor/Modal.tsx

```diff
@@ -234,3 +234,34 @@ const InnerForm: React.FC<InnerFormProps> = ({
     </Box>
   )
 }
+
+interface OnCompleteRedirModalProps {
+  onClick: () => void
+  redirectTo: string
+  show: boolean
+}
+
+export const OnCompleteRedirModal: React.FC<OnCompleteRedirModalProps> = props => {
+  const { onClick, redirectTo, show } = props
+
+  return (
+    <Modal
+      title="Set up with app"
+      onClose={onClick}
+      show={show}
+      hideCloseButton={true}
+      FixedButton={
+        <Button onClick={onClick} width="100%">
+          OK
+        </Button>
+      }
+    >
+      <Serif size="3t" color="black60">
+        You’ve successfully set up two-factor authentication!
+      </Serif>
+      <Serif mt={2} size="3t" color="black60">
```

Comment:
We're fazing out `Sans / Serif` in favor of just `Text`, but since all of these sorts of items will eventually need some new designs I think this is fine for now.

Reply:
okay.